### PR TITLE
Fix for my.contabo.com

### PR DESCRIPTION
**Pull request by ModProg**
I moved this from the Issue comment to a PR:
https://github.com/darkreader/darkreader/issues/4752#issuecomment-932731732

The configuration page my.contabo.com also had some issues, I added this to improve it:
```css
================================

my.contabo.com

CSS
#mainmenu {
    background-color: rgb(0, 0, 111) !important;
    border-radius: 20px 0px 0px 20px !important;
    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
}
#mainmenu > div {
    background-image: none !important;
}
#submenu {
    background-color: #977122 !important;
    border-radius: 20px 0px 0px 20px !important;
    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
}
#submenu > div {
    background-image: none !important;
}
.button {
    background-image: linear-gradient(#126382 49%, #0a4f6a 51%);
}
.ui-button:hover{
    background-image: linear-gradient(#807e7e 49%, #717171 51%) !important;
    color: white !important;
}
.ui-button {
    background-image: linear-gradient(#515151 49%, #5e5e5e 51%) !important;
    color: white !important;
}
```

This is how it looks with the fix:
grafik
grafik

This is how it looked before:
grafik
grafik

### CODE REVIEW

**Comment by ModProg** [comment]
Not sure if this is the best way to implement this, but the problem are the background images, so I replaced them with css.

**Comment by kizule** [comment]
+1.

Maybe we should fix and log in?

slika

**Comment by ModProg** [comment]
oh yeah forgot about that thing :D

**Comment by ModProg** [comment]
I only achieved this:


```css
================================

my.contabo.com/account/login

CSS
input[type="image"] {
    background-image: linear-gradient(#126382 49%, #0a4f6a 51%);
    border-radius: 5px
}
div {
    background-image: none !important;
}
```

I don't know how to fix this fully, so I'll keep it at this. 
If someone knows how to feel free to complete this.

**Comment by ModProg** [comment]
I only achieved this:
grafik

```css
================================

my.contabo.com/account/login

CSS
input[type="image"] {
    background-image: linear-gradient(#126382 49%, #0a4f6a 51%);
    border-radius: 5px
}
div {
    background-image: none !important;
}
```

I don't know how to fix this fully, so I'll keep it at this. 
If someone knows how to feel free to complete this.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -9154,6 +9154,39 @@ CSS
 
 ================================
 
+my.contabo.com
+
+CSS
+#mainmenu {
+    background-color: rgb(0, 0, 111) !important;
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9163]: Cosmetic change?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -9154,6 +9154,39 @@ CSS
 
 ================================
 
+my.contabo.com
+
+CSS
+#mainmenu {
+    background-color: rgb(0, 0, 111) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9161]:
```suggestion
    background-color: var(--darkreader-neutral-background) !important;
```

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -9154,6 +9154,39 @@ CSS
...
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
+}
+#mainmenu > div {
+    background-image: none !important;
+}
+#submenu {
+    background-color: #977122 !important;
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9171]: Cosmetic change?

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -9154,6 +9154,39 @@ CSS
...
+#mainmenu {
+    background-color: rgb(0, 0, 111) !important;
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
+}
+#mainmenu > div {
+    background-image: none !important;
+}
+#submenu {
+    background-color: #977122 !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9169]: Can this be implemented along the lines of:
```suggestion
    background-color: ${gray} !important;
```

File: src/config/dynamic-theme-fixes.config [hunk 5]
```diff
@@ -9154,6 +9154,39 @@ CSS
...
 
+my.contabo.com
+
+CSS
+#mainmenu {
+    background-color: rgb(0, 0, 111) !important;
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
+}
+#mainmenu > div {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9165]:
```suggestion
#mainmenu > div,
#submenu > div, {
```

File: src/config/dynamic-theme-fixes.config [hunk 6]
```diff
@@ -9154,6 +9154,39 @@ CSS
...
+    background-image: none !important;
+}
+#submenu {
+    background-color: #977122 !important;
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
+}
+#submenu > div {
+    background-image: none !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9175]:
```suggestion
```
See suggestion above.

File: src/config/dynamic-theme-fixes.config [hunk 7]
```diff
@@ -9154,6 +9154,39 @@ CSS
...
+.button {
+    background-image: linear-gradient(#126382 49%, #0a4f6a 51%);
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9177]: Ideally we leave gradients as is. Is their anything wrong with it?

File: src/config/dynamic-theme-fixes.config [hunk 8]
```diff
@@ -9154,6 +9154,39 @@ CSS
...
+}
+.ui-button:hover{
+    background-image: linear-gradient(#807e7e 49%, #717171 51%) !important;
+    color: white !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9181]: Seems like a duplicate `color`(from the non`:hover`)

File: src/config/dynamic-theme-fixes.config [hunk 9]
```diff
@@ -9154,6 +9154,39 @@ CSS
...
+}
+.ui-button {
+    background-image: linear-gradient(#515151 49%, #5e5e5e 51%) !important;
+    color: white !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9185]:
```suggestion
    color: var(--darkreader-neutral-text) !important;
```

File: src/config/dynamic-theme-fixes.config (hunk 7 above)
Comment (ModProg) [reply src/config/dynamic-theme-fixes.config:9177]: it was an image befor, I replaced it with a gradiant.

File: src/config/dynamic-theme-fixes.config (hunk 8 above)
Comment (ModProg) [reply src/config/dynamic-theme-fixes.config:9181]: that is true, I'll remove it

File: src/config/dynamic-theme-fixes.config (hunk 7 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:9177]: Just remove the background image that should be good enough.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (ModProg) [reply src/config/dynamic-theme-fixes.config:9163]: this tries to match the original, because it replaced the image that was there befor it.

**Comment by ModProg** [comment]
If you think we should remove all cosmetic things we could just do this:
```css

================================

my.contabo.com

CSS
#mainmenu > div, 
#submenu > div {
    background-image: none !important;
}
.button {
    background-image: none !important;
}
.ui-button {
    background-image: none !important;
    color: var(--darkreader-neutral-text) !important;
}
```

grafik
grafik

**Comment by Gusted** [comment]
> If you think we should remove all cosmetic things we could just do this:

The config shouldn't have any kind of cosmetic fix as these are opinionated and we like to keep the config as non-opinionated as possible.

**Comment by ModProg** [comment]
> > If you think we should remove all cosmetic things we could just do this:
> 
> The config shouldn't have any kind of cosmetic fix as these are opinionated and we like to keep the config as non-opinionated as possible.

I wasn't trying to fix anything cosmetic, I was trying to stay similar to the original. But I'll update my pr to the minimal config then.

**Comment by Gusted** [comment]
> I wasn't trying to fix anything cosmetic, I was trying to stay similar to the original. But I'll update my pr to the minimal config then.

While in cases like these it makes sense to have that, we have to ensure a overall "uniform" if we allow x case, why not allow case y and what kind of rules/criteria should apply to only allow case x.

File: src/config/dynamic-theme-fixes.config [hunk 10]
```diff
@@ -9154,6 +9154,23 @@ CSS
 
 ================================
 
+my.contabo.com
+
+CSS
+#mainmenu > div, 
+#submenu > div {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9161]:
```suggestion
#submenu > div,
.button,
.ui-button {
```

File: src/config/dynamic-theme-fixes.config [hunk 11]
```diff
@@ -9154,6 +9154,23 @@ CSS
...
+    background-image: none !important;
+}
+.button {
+    background-image: none !important;
+}
+.ui-button {
+    background-image: none !important;
+    color: var(--darkreader-neutral-text) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9170]:
```suggestion
.ui-button {
    color: var(--darkreader-neutral-text) !important;
}
```

**Review by Gusted — COMMENTED** [review]
Just to make it a little more tidier.

**Review by Gusted — APPROVED** [review]

**Pull request merged**